src/main/java/uk/gov/companieshouse/insolvency/data/config/ApplicationConfig.java

```diff
@@ -23,9 +37,33 @@ EnvironmentReader environmentReader() {
         return new EnvironmentReaderImpl();
     }
 
+    @Bean
+    @Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
+    public InternalApiClient internalApiClient() {
+        return ApiSdkManager.getPrivateSDK();
+    }
+
+    /**
+     * mongoCustomConversions.
+     * @return MongoCustomConversions.
+     */
     @Bean
     public MongoCustomConversions mongoCustomConversions() {
-        return new MongoCustomConversions(List.of(insolvencyConverter));
+        GenerateEtagUtil.generateEtag();
```

Comment:
Why is this here. when you're not using the etag value

Reply:
Changes done